### PR TITLE
[PREVIEW] - Saucelabs

### PR DESCRIPTION
**Pull request by rramisetty**

### CODE REVIEW

File: Jenkinsfile_nightly [hunk 1]
```diff
@@ -37,6 +37,17 @@ withNightlyPipeline(type, product, component) {
     enableSlackNotifications(channel)
     loadVaultSecrets(secrets)
 
+    enableCrossBrowserTest(120)
```
[the remark this answers is not in the document]
Comment (alectronic0) [reply Jenkinsfile_nightly]: as it a part of the setup enableing section we have with 
```
enableSlackNotifications(channel)
loadVaultSecrets(secrets)
// more enabling things
````

File: Jenkinsfile_parameterized [hunk 2]
```diff
@@ -41,7 +41,7 @@ withPipeline(type , product, component) {
 //    enableApiGatewayTest(15)
 //    enableFullFunctionalTest(30)
 //    enableMutationTest(120)
-//    enableCrossBrowserTest(120)
+ //     enableCrossBrowserTest(120)
```
Comment (alectronic0) [inline Jenkinsfile_parameterized]: clear your change from here
this is a pointless commit

File: package.json [hunk 3]
```diff
@@ -29,7 +29,7 @@
     "test:smoke": "webdriver-manager update --versions.chrome 2.40 && protractor ./test/e2e/config/smoke_conf.js",
     "test:functional": "webdriver-manager update --versions.chrome 2.40 && protractor ./test/e2e/config/conf.js",
     "test:functional:local": "webdriver-manager update --versions.chrome 2.40 && protractor ./test/e2e/config/conf.js --local",
-    "test:crossbrowser": "echo 'not implemented'",
+    "test:crossbrowser": "protractor ./test/e2e/config/saucelabs.conf.js",
```
Comment (alectronic0) [inline package.json]: nice :D

File: test/e2e/config/saucelabs.conf.js [hunk 4]
```diff
@@ -0,0 +1,154 @@
+const chai = require('chai');
```
Comment (alectronic0) [inline test/e2e/config/saucelabs.conf.js]: maybe call this cross browser config rather than sauselab

Comment (rramisetty) [reply test/e2e/config/saucelabs.conf.js]: sure @alectronic0  will update now

File: package.json (hunk 3 above)
Comment (rramisetty) [reply package.json]: thank you

File: Jenkinsfile_parameterized (hunk 2 above)
Comment (rramisetty) [reply Jenkinsfile_parameterized]: i have already resolved this

File: Jenkinsfile_nightly (hunk 1 above)
[the remark this answers is not in the document]
Comment (rramisetty) [reply Jenkinsfile_nightly]: those are not implemented yet .. so commented out but added them

**Pull request merged**